ansible/roles/projects/ovn-kubernetes/undeploy/tasks/main.yml

```diff
@@ -1,8 +1,11 @@
 ---
 - name: Undeploy ovn-kubernetes components
-  k8s:
-    src: "{{ artifacts }}/{{ item }}.yaml"
-    state: absent
-    kubeconfig: "{{ kubeconfig }}"
-    context: "{{ k8s_context }}"
+  vars:
+    kubernetes_object_file: "{{ artifacts }}/{{ ovn_kubernetes_object }}.yaml"
+  include_role:
+    name: projects-common/undeploy-kubernetes-object-from-file
```

Comment:
This does not seems to be related to the commit it was added on (deploy multus). I believe this should be moved to its relevant commit

Reply:
This is needed to not redundant the code in the same commit